objects/cash_register.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+# here's my cash register
+class CashRegister
+  def initialize
+    @owed = 0.0
+  end
+
+  def total
+    '%.2f' % @owed
+  end
+
+  def purchase(price)
+    @owed = price + @owed
+  end
+
+  def pay(amount_paid)
+    change = amount_paid - @owed.to_f
```

Comment:
If you did `@owed.to_f - amount_paid` I don't think you'd need the `.abs` call below.